examples/lib/intermediateHtmlStepUi.js

```diff
@@ -4,7 +4,6 @@ var urlHash = require('./urlHash.js'),
 function IntermediateHtmlStepUi(_sequencer, step, options) {
   function stepUI() {
     return '<div class="row insertDiv collapse">\
-        <div class="col-md-6 col-md-offset-2" style="margin-top:5%">\
           <section class="panel panel-primary .insert-step">\
             <button class="btn btn-default close-insert-box"><i class="fa fa-times" aria-hidden="true"></i> Close</button>\
```

Comment:
@HarshKhandeparkar remove this line. Because you are appending the close option for two times.

Reply:
Sorry that's not the line. Remove this line
<button class="btn btn-default close-insert-box"><i class="fa fa-times" aria-hidden="true"></i> Close</button>\

Reply:
Jywarren asked me to do that

Reply:
ok remove the above line which I mentioned.

Reply:
Jywarren asked me to add a close button inside the box itself.

Reply:
Oh but it looks odd if we have two buttons for the same option.

Reply:
Most of the times, the other btn will be unnoticed. If by mistake someone opens the box, the btn that is used to open the dialog, can be used to close it.